modules/dataframeqt/src/dataframetableprocessorwidget.cpp

```diff
@@ -48,23 +48,19 @@ namespace inviwo {
 DataFrameTableProcessorWidget::DataFrameTableProcessorWidget(Processor* p)
     : ProcessorWidgetQt(p), tableview_(nullptr) {
 
-    this->setParent(utilqt::getApplicationMainWindow());
+    setParent(utilqt::getApplicationMainWindow());
     setMinimumSize(32, 32);
     QObject::setProperty("bgType", "window");
 
-    tableview_ = tableview_ptr(new DataFrameTableView(this), [&](DataFrameTableView* c) {
-        layout()->removeWidget(c);
```

Comment:
did you test this? Used to be a problem on destruction before.

Reply:
I've been using it for some time without any issue... and I don't see any logical reasons this would be needed.

Reply:
Im not using any unique ptr at all but letting qt handle the lifetime...